Makefile

```diff
@@ -0,0 +1,24 @@
+# Project packages.
+PACKAGES=$(shell go list ./... | grep -v /vendor/)
+
+# Flags passed to `go test`
+TESTFLAGS ?= 
```

Comment:
perhaps we should use `-v` (verbose) as default?

Reply:
I don't like verbose as default. If we want verbose in the CI we should set it, verbose on the command line can mean you hide useful output.

Reply:
I'm wondering how many go developers would use `make test`, instead of `go test` (if the codebase doesn't require special steps).

It'd probably be useful to have a `go mod tidy` step somewhere in CI though 🤔

Reply:
If there's a makefile I expect there to be a `make test`. If I want to know if the project does require special steps, I'll check the makefile.

Reply:
Yes, that makes sense as well. Initially I tried to avoid adding a Makefile, but I agree that for the coverage, it's useful, as that's a bit more typing to do.